test-utils/test-utils-core/src/main/java/uk/gov/justice/services/test/utils/core/random/NiNumberGenerator.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.services.test.utils.core.random;
+
+import static java.lang.String.format;
+
+public class NiNumberGenerator implements Generator<String> {
```

Comment:
This one should probably not be in the microservice framework... lets leave it here for not, but it needs to move